modules/rdsh/ra_rdsh_autoscale_internal_lb.template.cfn.yaml

```diff
@@ -298,9 +298,11 @@ Resources:
   AutoScalingGroup:
     CreationPolicy:
       ResourceSignal:
-        Count: !Ref DesiredCapacity
+        Count: 1
         Timeout: PT85M
     Properties:
+      Cooldown: 0
+      DefaultInstanceWarmup: 0
```

Comment:
Seems like `DefaultInstanceWarmup` isn't supported by Cloudformation:

https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html#cfn-as-group-defaultinstancewarmup

Reply:
It is though! 

* https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-as-group.html#cfn-as-group-defaultinstancewarmup

It's just recently added and lots of things haven't caught up yet.

Reply:
Ok.